services/file_delete.proto

```diff
@@ -29,16 +36,29 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * Delete the given file. After deletion, it will be marked as deleted and will have no contents.
- * But information about it will continue to exist until it expires. A list of keys was given when
- * the file was created. All the top level keys on that list must sign transactions to create or
- * modify the file, but any single one of the top level keys can be used to delete the file. This
- * transaction must be signed by 1-of-M KeyList keys. If keys contains additional KeyList or
- * ThresholdKey then 1-of-M secondary KeyList or ThresholdKey signing requirements must be meet.
+ * Mark a file as deleted and remove its content from network state.
+ *
+ * The metadata for a deleted file SHALL be retained at least until the
+ * expiration time for the file is exceeded.<br/>
+ * On completion, the identified file SHALL be marked `deleted`.<br/>
+ * On completion, the identified file SHALL have an empty `contents` array.<br/>
+ * This transaction SHALL be final and irreversible.
+ *
```

Comment:
Add a new paragraph:
If the `keys` list for the identified file is an empty `KeyList` (because this file was
 * previously created or updated to have an empty `KeyList`), then the file is considered
 * immutable and this message WILL fail as UNAUTHORIZED.

Reply:
Added.